moto/ec2/models.py

```diff
@@ -5986,6 +5993,103 @@ def modify_transit_gateway(self, transit_gateway_id, description=None, options=N
         return transit_gateway
 
 
+class TransitGatewayRouteTable(TaggedEC2Resource):
+
+    def __init__(
+        self,
+        backend,
+        transit_gateway_id,
+        tags=[],
+        default_association_route_table=False,
+        default_propagation_route_table=False,
+    ):
+        self.ec2_backend = backend
+        self.id = random_transit_gateway_route_table_id()
+        self.transit_gateway_id = transit_gateway_id
+
+        self._created_at = datetime.utcnow()
+
+        self.default_association_route_table = default_association_route_table
+        self.default_propagation_route_table = default_propagation_route_table
+        self.state = "available"
+        self.add_tags(tags or {})
+
+    @property
+    def physical_resource_id(self):
+        return self.id
+
+    @property
+    def create_time(self):
+        return iso_8601_datetime_with_milliseconds(self._created_at)
+
+
+class TransitGatewayRouteTableBackend(object):
+    def __init__(self):
+        self.transit_gateways_route_tables = {}
+        super(TransitGatewayRouteTableBackend, self).__init__()
+
+    def create_transit_gateway_route_table(
+        self,
+        transit_gateway_id,
+        tags=[],
+        default_association_route_table=False,
+        default_propagation_route_table=False
+    ):
+        transit_gateways_route_table = TransitGatewayRouteTable(
+            self,
+            transit_gateway_id=transit_gateway_id,
+            tags=tags,
+            default_association_route_table=default_association_route_table,
+            default_propagation_route_table=default_propagation_route_table
+        )
+        self.transit_gateways_route_tables[transit_gateways_route_table.id] = transit_gateways_route_table
+        return transit_gateways_route_table
+
+    def get_all_transit_gateway_route_tables(self, filters):
+        transit_gateway_route_tables = self.transit_gateways_route_tables.values()
+
+        if filters is not None:
+            if filters.get("default-association-route-table") is not None:
```

Comment:
nitpick: We could parameterize the code below (wrapping it in a `for` loop) to reduce the duplication a bit. (not critical, though)